scrapyjs/request.py

```diff
@@ -38,21 +42,48 @@ def __init__(self,
             splash_meta['splash_url'] = splash_url
         if splash_headers is not None:
             splash_meta['splash_headers'] = splash_headers
+        if dont_process_response:
+            splash_meta['dont_process_response'] = True
+        else:
+            splash_meta.setdefault('magic_response', magic_response)
+        if dont_send_headers:
+            splash_meta['dont_send_headers'] = True
+
+        if session_id is not None:
```

Comment:
Now I see why it is done this way, and I think it's logical, please ignore the comment :)